ci-setups/php-github.md

```diff
@@ -0,0 +1,144 @@
+# Setup Continuous Integration (CI) on Github for PHP projects
+
+## Introduction
+
+Continuous Integration (CI) in GitHub is a software development practice that involves automatically integrating code changes from multiple developers into a shared repository on a frequent and ongoing basis. The main goal of CI is to detect and address integration issues early in the development process, ensuring that code changes are tested, validated, and merged into the main codebase as smoothly as possible.
+
+Github provides [Github Actions](https://github.com/features/actions) that can be used to create [workflows](https://docs.github.com/en/actions/using-workflows/about-workflows) that can be used for performing coding standards checks for various programming languages, including PHP.
+
+This page will walk you through the steps of setting up a Github Action workflow for performing PHP codings standards checks with PHPCS and Phan for linting and analysis of code.
+
+## Setting up Github Actions for project
+
+### Pre-requisites
+
+- An existing or new repository with the PHP project
+- All required configurations done as per [Enforcing tools and config](https://github.com/OsmosysSoftware/dev-standards/blob/main/coding-standards/php.md#enforcing-tools-and-config)
+
+### Creating a workflow
+
+A workflow is a configurable automated process that will run one or more jobs. Workflows are defined by a YAML file checked in to your repository and will run when triggered by an event in your repository. These files are defined in `.github/workflows`.
+
+There are two methods that can be followed for creating a workflow:
+
+#### Method 1: Using Github UI
+
+1. Open your project repository and click on the **Actions** tab
+
+    ![Github Actions Tab](assets/php-github-actions-tab.png)
+
+2. A new page **Choose a workflow** will open, click on the **set up a workflow yourself** link
+
+    ![Github Choose a workflow](assets/php-github-choose-workflow.png)
+
+    > Note: In case the project already has some other workflows set up, click on the **New workflow** button on the left side. This will take you to the above page.
+
+3. A new page will open which will by default create a new file `.github/workflows/main.yml` in edit mode. For the purposes of this guide, we will rename the file to `php-lint.yml`, though it can be named as seemed fit.
+
+    ![Github New workflow](assets/php-github-new-workflow.png)
+
+4. Add the following contents to this file:
+
+    ```yaml
+    name: PHP Coding Standards
+    on:
+      push:
+        branches: [ main ]
```

Comment:
Add example for multiple branches here.

Reply:
This has been addressed in the most recent commit.